data/simpy.gram

```diff
@@ -438,18 +439,25 @@ star_target[expr_ty]:
     | star_atom
 star_atom[expr_ty]:
     | a=NAME { set_expr_context(p, a, Store) }
+    | '(' a=[star_target !','] ')' { set_expr_context(p, a, Store) }
     | '(' a=[star_targets_seq] ')' { _Py_Tuple(a, Store, EXTRA) }
     | '[' a=[star_targets_seq] ']' { _Py_List(a, Store, EXTRA) }
 
+ann_assign_target[expr_ty]:
+    | a=t_primary '.' b=NAME !t_lookahead { _Py_Attribute(a, b->v.Name.id, Store, EXTRA) }
+    | a=t_primary b=slicing !t_lookahead { _Py_Subscript(a, b, Store, EXTRA) }
+    | a=NAME { set_expr_context(p, a, Store) }
+
 del_targets[asdl_seq*]: a=','.del_target+ [','] { a }
 del_target[expr_ty]:
     | a=t_primary '.' b=NAME !t_lookahead { _Py_Attribute(a, b->v.Name.id, Del, EXTRA) }
     | a=t_primary b=slicing !t_lookahead { _Py_Subscript(a, b, Del, EXTRA) }
     | del_t_atom
 del_t_atom[expr_ty]:
     | a=NAME { set_expr_context(p, a, Del) }
-    | '(' b=[del_targets] ')' { _Py_Tuple(b, Del, EXTRA) }
-    | '[' b=[del_targets] ']' { _Py_List(b, Del, EXTRA) }
+    | '(' a=[del_target !','] ')' { set_expr_context(p, a, Del) }
```

Comment:
I don't see why you need the `!','` here, since you already have `')'` following.